### PR TITLE
MAINT: Update the Call for Contributions section

### PR DESCRIPTION
**Pull request by InessaPawson**
Expanding the section to provide info about the types of contribution that exist within the NumPy project and points of contact with the community coordinators and other contributors.

### CODE REVIEW

**Review by melissawm — APPROVED** [review]
LGTM, @InessaPawson ! We could just link to [numpy.org/contribute](numpy.org/contribute) but having this information straight on the README does seem friendlier to me.

File: README.md [hunk 1]
```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
...
-NumPy appreciates help from a wide range of different backgrounds.
-Work such as high level documentation or website improvements are valuable
-and we would like to grow our team with people filling these roles.
-Small improvements or fixes are always appreciated and issues labeled as easy
-may be a good starting point.
-If you are considering larger contributions outside the traditional coding work,
-please contact us through the mailing list.
+The NumPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated. (Issues labeled as easy may be a good starting point.) If you are considering larger contributions to the source code, please contact us through the [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion) first.
```
Comment (mattip) [inline README.md:35]: "easy" -> "good first issue", fix line length
```suggestion
Small improvements or fixes are always appreciated. (Issues labeled as "good
first issue" may be a good starting point.) If you are considering larger
contributions to the source code, please contact us through the [mailing
list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
```

File: README.md [hunk 2]
```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
...
+Small improvements or fixes are always appreciated. (Issues labeled as easy may be a good starting point.) If you are considering larger contributions to the source code, please contact us through the [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
+
+Writing code isn’t the only way to contribute to NumPy. You can also: 
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve our website numpy.org
+- develop graphic design for our brand assets and promotional materials
+- translate website content
+- serve as a community coordinator
```
Comment (mattip) [inline README.md:44]:
```suggestion
- help with outreach and onboard new contributors
```

File: README.md [hunk 3]
```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
...
+- write grant proposals and help with other fundraising efforts
+
+If you’re unsure where to start or how your skills fit in, reach out! You can ask on the mailing list or here, on GitHub, by opening a new issue or leaving a comment on a relevant issue that is already open.
```
Comment (mattip) [inline README.md:47]: line length
```suggestion
If you’re unsure where to start or how your skills fit in, reach out! You can
ask on the mailing list or here, on GitHub, by opening a new issue or leaving a
comment on a relevant issue that is already open.
```

File: README.md [hunk 4]
```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
...
+
+Those are our preferred channels (open source is open by nature), but if you’d like to speak to us in private first, contact our community coordinators at numpy-team@googlegroups.com or on Slack (write numpy-team@googlegroups.com for an invite).
```
Comment (mattip) [inline README.md:49]:
```suggestion
Our preferred channels of communication are all public, but if you’d like to
speak to us in private first, contact our community coordinators at
numpy-team@googlegroups.com or on Slack (write numpy-team@googlegroups.com for
an invite).
```

File: README.md [hunk 5]
```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
...
+
+We also have a biweekly community call, details of which are announced on the mailing list. You are very welcome to join. 
+
+If you are new to contributing to open source, we highly recommend reading [this guide](https://opensource.guide/how-to-contribute/).
```
Comment (mattip) [inline README.md:53]: formatting, and switch "we highly recommend" to a ten word summary of what they will find at the link
```suggestion
We also have a biweekly community call, details of which are announced on the
mailing list. You are very welcome to join. 

If you are new to contributing to open source, [this
guide](https://opensource.guide/how-to-contribute/) helps explain why, what,
and how to successfully get involved.
```

**Review by mattip — COMMENTED** [review]
Thanks. A few suggestions

File: README.md (hunk 1 above)
Comment (stefanv) [reply README.md:35]: Also link to those issues:

https://github.com/numpy/numpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Review by stefanv — COMMENTED** [review]
Thank you @InessaPawson and +1 on all @mattip's suggestions!

File: README.md [hunk 6]
```diff
@@ -30,13 +30,39 @@ Tests can then be run after installation with:
...
-Work such as high level documentation or website improvements are valuable
-and we would like to grow our team with people filling these roles.
-Small improvements or fixes are always appreciated and issues labeled as easy
-may be a good starting point.
-If you are considering larger contributions outside the traditional coding work,
-please contact us through the mailing list.
+The NumPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated. (Issues labeled as "good
+first issue" may be a good starting point.) If you are considering larger
```
Comment (charris) [inline README.md:36]: Maybe use a ";" instead of a period and a parenthesized sentence.

File: README.md [hunk 7]
```diff
@@ -30,13 +30,39 @@ Tests can then be run after installation with:
...
+contributions to the source code, please contact us through the [mailing
+list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
+
+Writing code isn’t the only way to contribute to NumPy. You can also: 
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve our website numpy.org
```
Comment (Qiyu8) [inline README.md:44]: Maybe add website source code repository instead of numpy.org.

**Comment by mattip** [comment]
Thanks @InessaPawson. Let's put this in and, if there is a need, iterate in new PRs.

**Pull request merged**